config/initializers/ab_tests.rb

```diff
@@ -58,4 +58,24 @@ def self.all
   ) do |service_provider:, session:, user:, user_session:, **|
     document_capture_session_uuid_discriminator(service_provider:, session:, user:, user_session:)
   end.freeze
+
+  RECAPTCHA_SIGN_IN = AbTest.new(
+    experiment_name: 'reCAPTCHA at Sign-In',
+    should_log: Set.new(
+      [
+        'Email and Password Authentication',
+        'IdV: doc auth verify proofing results',
+        'reCAPTCHA verify result received',
+        :user_suspension_confirmed,
+        'User Suspension: Suspended',
+      ],
```

Comment:
save a little indentation potentially
```suggestion
    should_log: [
      'Email and Password Authentication',
      'IdV: doc auth verify proofing results',
      'reCAPTCHA verify result received',
      :user_suspension_confirmed,
      'User Suspension: Suspended',
    ].to_set,
```

Reply:
Yeah I like that. Originally I tried `Set.new([`, but Rubocop doesn't take a liking to it. Kinda a shame, I dunno if there's an exception for that? It happens a lot elsewhere, seems like `(` + `[` should be allowed as an exception.

Reply:
Updated to use `Array#to_set` in d15bf35